README.markdown

```diff
@@ -377,6 +377,22 @@ Calling the class or definition from outside the current module will fail. For e
 
   *Type*: statement
 
+* `pw_hash`: Hashes a password using the crypt function. Provides a hash usable on most POSIX systems.
+
+  The first argument to this function is the password to hash. If it is undef or an empty string, this function returns undef.
+
+  The second argument to this function is a specifier of which hash type to use. Valid hash types are:
+
+  |Specifier|Hash type            |
+  |---------|---------------------|
+  |1        |MD5                  |
+  |5        |SHA-256              |
+  |6        |SHA-512 (recommended)|
```

Comment:
There's a difference between "you shouldn't use this" and "we recommend you to use the strongest one".